Return interactive pareto plot via bokeh-json

abbl-DTI opened the pull request:
Update to the return for process_result when optimizer.n_objectives > 1. 
This update is dependent on ProcessOptimizer = 7.5
This PR closes #48

abbl-DTI committed:
Changes made so for Pareto plotting

abbl-DTI committed:
Update 'requirements-freeze.txt'

abbl-DTI committed:
typing-extensions version updated

abbl-DTI committed:
Update to latest ProcessOptimizer

langdal commented:
It should be possible to use fixed dependencies of ProcessOptimizer by installing using pip install ProcessOptimizer[browniebee] as described in novonordisk-research/ProcessOptimizer#108

langdal committed:
Use ProcessOptimizer with locked deps

Use the Brownie Bee specific package of ProcessOptimizer.
Update frozen requirements according to README

langdal commented:
@abbl-DTI Tests fail in this branch because a sub-module in ProcessOptimizer is missing from the distribution package. It looks like the module "model_systems" is not included in the distribution. Looking at the ProcessOptimizer repo it looks like that module has been added recently and might need to be added to the build script (setup.py)?

langdal commented:
@abbl-DTI I just noticed that you already created an issue in PO for this :)
novonordisk-research/ProcessOptimizer#124

btw. I have added a commit that use the fixed deps. version of PO

langdal committed:
Merge branch 'main' into Return-pareto-plot-as-bokeh-json
